app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="contestants-index">
+  <h1>All Contestants</h1>
+
+  <section class="contestant-list">
+    <div class="list-of-contestants">
+      <h2> Contestants:</h2>
+      <% @contestants.each do |contestant| %>
+        <div id="contestant-<%= contestant.id %>">
+          <p>Name: <%= contestant.name %></p>
+          <p>Projects: <%= contestant.projects.map { |project| project.name }.join(', ') %></p>
```

Comment:
Try to avoid using `map` in your view. I would consider getting all of the project names data logic, and so I would recommend making a Contestant model method called `project_names`